### PR TITLE
Update ENS files with latest versions

### PR DESCRIPTION
**Pull request by i-stam**
This tests the latest versions of ENS related files that got updated due to a bug found.

### CODE REVIEW

**Comment by mischat** [comment]
@nostdm it would be nice to have a look at the echnida tests here, or if you could explain then so we learn. It is good that they detected a change as our tests didn't detech any change to the ENS APIs we current utilise ...

**Comment by mischat** [comment]
@i-stam ... this looks good to me, but would like the echnida tests to pass.

**Comment by nostdm** [comment]
Yep I fixed the tests, it was just a tooling error

**Comment by mischat** [comment]
Looking at the warnings coming out of the compiler, they don't even make sense. I have looked at around 5 of them in the ENSRegistry and the PublicResolver, all of them were wrong. There was no shadowing .... this sucks, how hard would be to make a pre on the 0.5.X compiler!

**Comment by mischat** [comment]
@nostdm do you want to double check this ?

File: contracts/externals/ens/profiles/TextResolver.sol [hunk 1]
```diff
@@ -34,4 +34,4 @@ contract TextResolver is ResolverBase {
     function supportsInterface(bytes4 interfaceID) public pure returns(bool) {
         return interfaceID == TEXT_INTERFACE_ID || super.supportsInterface(interfaceID);
     }
-}
+}
```
Comment (nostdm) [inline contracts/externals/ens/profiles/TextResolver.sol:37]: Some files are missing new lines at the end

File: test/wallet/wallet_suite_test.go [hunk 2]
```diff
@@ -99,7 +99,7 @@ var _ = BeforeEach(func() {
 })
 
 var allPassed = true
-var currentVersion = "3.1.0"
+var currentVersion = "3.1.1"
```
Comment (nostdm) [inline test/wallet/wallet_suite_test.go:102]: BTW I already tagged 3.1.1 we could either re-tag it or make it 3.1.2

File: contracts/externals/ens/profiles/TextResolver.sol (hunk 1 above)
Comment (mischat) [reply contracts/externals/ens/profiles/TextResolver.sol:37]: we should add them in ... what does our linter think about this ?

File: test/wallet/wallet_suite_test.go (hunk 2 above)
Comment (i-stam) [reply test/wallet/wallet_suite_test.go:102]: ah yeah, just noticed it, @mischat  is the right person to talk about tagging, I am not opinionated.

Comment (mischat) [reply test/wallet/wallet_suite_test.go:102]: on the tagging, can we please stop tagging things and have conversations about them. The tags are basically within our bug bounty, and as a result, i don't want to just tag random things. It is OK to get a version in the code, and to tag the actual repo when we are happy with the state of master.

In short, let's have conversations before we tag things, i thought you updated the currentVersion variable in the repo (which i saw) but I didn't now that you actually tagged it. @nostdm

File: contracts/externals/ens/profiles/TextResolver.sol (hunk 1 above)
Comment (i-stam) [reply contracts/externals/ens/profiles/TextResolver.sol:37]: I think that the formating tool does not check the external files, maybe we should add them. To me it is irrelevant and not so important.

File: test/wallet/wallet_suite_test.go (hunk 2 above)
Comment (nostdm) [reply test/wallet/wallet_suite_test.go:102]: OK, I thought this applied to minor/major versions and not the patch versions

File: contracts/externals/ens/profiles/TextResolver.sol (hunk 1 above)
Comment (nostdm) [reply contracts/externals/ens/profiles/TextResolver.sol:37]: I explicitly ignored them, didn't want to format external code. But you can add them to format.sh if you think it's a good idea

**Pull request merged**